src/backend/vulkan/src/lib.rs

```diff
@@ -61,9 +61,10 @@ lazy_static! {
         vec![
             DebugUtils::name(),
             DebugReport::name(),
+            CStr::from_bytes_with_nul(b"VK_KHR_get_physical_device_properties2\0").unwrap(),
         ]
     } else {
-        vec![]
+        vec![CStr::from_bytes_with_nul(b"VK_KHR_get_physical_device_properties2\0").unwrap()]
```

Comment:
let's put this string as a constant alongside the other extension names (e.g.`AMD_NEGATIVE_VIEWPORT_HEIGHT`) and re-use in here

Reply:
I tried that but was getting some inscrutable type error, need to try again when I'm not frustrated with vulkan's existance.